src/code-of-conduct.md

```diff
@@ -4,21 +4,21 @@ description: Be excellent to each other.
 toc: false
 ---
 
-As contributors and maintainers of the Dart project,
-we promise to extend courtesy and respect to everyone involved in this project
-regardless of gender, gender identity, sexual orientation, disability, age,
```

Comment:
I personally think this is actually quite important. We should call out various examples forms of harassment so that folk can't claim that their comment is 'out of scope' somehow. 

Another example of this is here: https://gist.github.com/Integralist/4e84af31c483d0f5a182, which I also like. I think upon occasion people appreciate having the specifics spelled out.

Reply:
I like being less specific, because you can never list all the categories. Someone wanted political leanings added, which I totally get, but I hesitated to pile on so many things that the details obscure the overall meaning.

I modeled this after the Flutter code of conduct, which is nicely general (to my eyes).